.env.example

```diff
@@ -0,0 +1,50 @@
+# Base Path Opensift: /businesses/create Firebase: /
+VUE_APP_PATH=/businesses/create
```

Comment:
Should this start with / or not (like previous one)?

Reply:
Fixed. It is different than other apps.